src/elf.rs

```diff
@@ -208,6 +214,10 @@ impl SectionBuilder {
                 shdr.sh_type = SHT_PROGBITS;
                 shdr.sh_flags |= (SHF_MERGE | SHF_STRINGS) as u64;
             },
+            SectionType::Data => {
+                shdr.sh_addralign = if self.exec { 0x10 } else if self.write { 0x8 } else { 1 };
```

Comment:
We may end up wanting an alignment hint via an optional but this is great for Bek